indico/modules/events/tracks/models/tracks.py

```diff
@@ -108,3 +108,11 @@ def __repr__(self):
 
     def can_delete(self, user):
         return self.event_new.can_manage(user) and not self.abstracts_accepted
+
+    def is_user_reviewer(self, user):
```

Comment:
Heh, looks like we added pretty much the same methods ;)

ee30b2d213754dabe9d0d287f91863bd255a2e6f (not in devel yet; it's in the branch @DavidAndreev, @mvidalgarcia and me worked on yesterday)

I think we should decide whether we want to check the reviewer/convener ACL role too or not and then remove it in all places where we currently do or add it to the places where we don't.

I'm tending towards not checking it since being in the reviewer list should be a strong enough criterion to consider someone a reviewer.

Either way, I'd add a `if user is None: return False` here - I doubt we even end up calling this if there's no user but in case we do there's no need to even fetch the reviewer/convener lists.

For the naming... I like yours more (never liked "can_convene" much, sorry @OmeGak ;)) - OTOH, `can_xxx` is what we use everywhere else even though it's technically not the correct name (since it's not checking whether the object can do something but the user passed to the method..)